### PR TITLE
refactor(util): Move unzip to util bundle

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR moves the `UnZip` class from `org.eclipse.kura.core.cloud` to `org.eclipse.kura.util` bundle.

**Related Issue:** This PR fixes/closes N/A

**Description of the solution adopted:** The utility `UnZip` class is moved to the `org.eclipse.kura.util.zip` package. This package is exported, so every bundle can use it. Moreover, some tests have been added.

### CODE REVIEW

File: kura/test/org.eclipse.kura.util.test/src/test/java/org/eclipse/kura/util/zip/UnZipTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package org.eclipse.kura.util.zip;
```
Comment (MMaiero) [inline kura/test/org.eclipse.kura.util.test/src/test/java/org/eclipse/kura/util/zip/UnZipTest.java:13]: copyright header

**Comment by MMaiero** [comment]
@mattdibi Can you please have a review of this PR?

**Comment by mattdibi** [comment]
One minor thing: in the TritonEncryptionUtils I added a check for the Zip magic number [here](https://github.com/eclipse/kura/blob/e62c1d77ff5fdcda96dc5f25994f19b5d04e9fdf/kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerEncryptionUtils.java#L122-L134). I think it should be moved in the UnZip class too. What do you think?

**Review by mattdibi — APPROVED** [review]
LGTM

Tested on Nvidia Jetson Nano: the Triton Server Service correctly decompress the archives containing the models. Likewise the Command Service correctly decompresses the archives uploaded to the target device.

**Comment by MMaiero** [comment]
There are tests failures on the commandCloudApp tests. Expected?

**Review by mattdibi — APPROVED** [review]
LGTM.

Re-run tests. Still passing.

**Pull request merged**